pgoapi/auth_ptc.py

```diff
@@ -27,89 +27,89 @@
 from future.standard_library import install_aliases
 install_aliases()
 
-import re
-import json
 import logging
 import requests
 
-from urllib.parse import parse_qs
+from urllib.parse import parse_qs, urlsplit
 from six import string_types
 
 from pgoapi.auth import Auth
 from pgoapi.utilities import get_time
 from pgoapi.exceptions import AuthException, InvalidCredentialsException
 
-from requests.exceptions import ConnectionError
+from requests.exceptions import RequestException
 
 class AuthPtc(Auth):
 
     PTC_LOGIN_URL = 'https://sso.pokemon.com/sso/login?service=https%3A%2F%2Fsso.pokemon.com%2Fsso%2Foauth2.0%2FcallbackAuthorize'
     PTC_LOGIN_OAUTH = 'https://sso.pokemon.com/sso/oauth2.0/accessToken'
     PTC_LOGIN_CLIENT_SECRET = 'w8ScCUXJQc6kXKw8FiOhd8Fixzht18Dq3PEVkUCP5ZPxtgyWsbTvWHFLm2wNY0JR'
 
-    def __init__(self):
+    def __init__(self, username=None, password=None):
         Auth.__init__(self)
 
         self._auth_provider = 'ptc'
 
         self._session = requests.session()
-        self._session.verify = True
+        self._session.headers = {'User-Agent': 'pokemongo/0 CFNetwork/758.5.3 Darwin/15.6.0'}
```

Comment:
You can if you create the `AuthPtc` object yourself and then modify `auth._session.headers`. Where would you want to set it? A `set_authentication` parameter?

At any rate, this is a saner default than `niantic` for a couple requests and `python requests` for the other (#163).